js-pkg/apps/tests/package.json

```diff
@@ -12,7 +12,7 @@
     "bun-types": "^0.5.7",
     "driftdb-internal": "*"
   },
-  "devDependencies": {
+  "optionalDependencies": {
```

Comment:
Why this change?

Reply:
I'm not sure. for whatever reason, the bun installscript doesn't seem to run correctly and causes npm to fail.